app/scripts/vueCore.js

```diff
@@ -269,11 +269,26 @@
               var unmappedCaseIDs = [];
 
               _.each(selectedCases, function(id) {
-                var caseUIDs = iViz.getCaseUID(selectionType, id);
-                if (caseUIDs.length === 0) {
-                  unmappedCaseIDs.push(id);
-                } else {
-                  selectedCaseUIDs = selectedCaseUIDs.concat(caseUIDs);
+                if (id) {
```

Comment:
why is this `if` condition required here

Reply:
Hm... after second thought, it shouldn't be there. But I remember I did encounter to the undefined issue. Remove for now.